src/app/public/src/modules/shared/widget.service.ts

```diff
@@ -74,6 +74,10 @@ export class HelpWidgetService {
     return Promise.resolve();
   }
 
+  public ready(): Promise<any> {
+    return BBHelpClient.ready();
+  }
+
   public executeWhenClientReady(callBack: any): Promise<any> {
     return BBHelpClient.ready()
```

Comment:
Maybe return `this.ready()`, so that you don't have duplicate references to the same method?